bundle/LegacyMapper/Configuration.php

```diff
@@ -180,6 +180,9 @@ public function onBuildKernel( PreBuildKernelEvent $event )
         // Multisite settings (PathPrefix and co)
         $settings += $this->getMultiSiteSettings();
 
+        // SiteAccess settings
+        $settings += $this->getSiteSettings();
```

Comment:
my point was that this line will overwrite the same settings coming from getMultiSiteSettings()